ds_judgements_public_ui/templates/pages/home.html

```diff
@@ -1,4 +1,4 @@
-{% extends "layouts/base.html" %}
+{% extends "layouts/base_no_breadcrumbs.html" %}
```

Comment:
We shouldn't need to create a copied layout just to remove the breadcrumbs. If you put the breadcrumbs in a `block` then here define the block as empty, it'll be empty on the homepage

Reply:
Alternatively given that breadcrumbs are built based on an array being present in the template variables, we would (ideally) just switch them off where that array is empty. That way in other views (like advanced search) where breadcrumbs aren't specified by the view it will also be breadcrumb-free, rather than having to remember to twiddle it in the template.